core-tools.markdown

```diff
@@ -0,0 +1,345 @@
+---
+title: Learn about the core tools
+page: core-tools
+isDocumentation: true
+---
+
+# Learn about the core tools
+
+In this guide we'll take a look at a few core tools that are installed
+with the Haskell toolchain, namely, `ghc`, `runghc` and `ghci`.
+These tools can be used to compile, interpret or explore Haskell programs.
+
+Note: if you installed your Haskell toolchain using the stack tool only
```

Comment:
See above, while I think this note is perfectly reasonable if we're not making assumptions about how things are installed, I think it would be a much better document if we just assumed an installation method.